deployment_report/model/viya_deployment_report.py

```diff
@@ -504,6 +520,25 @@ def get_sas_component_resources(self, component_name: Text, resource_kind: Text)
         except KeyError:
             return None
 
+    def get_cadence_data(self, resource: KubernetesResource) -> Optional[Text]:
```

Comment:
since this function is no longer returning data but just string value of the cadence name+version+release, perhaps this function should be renamed as "get_cadence_version" or something like that

Reply:
- renamed function to get_cadence_version